org.gnome.Hamster.yaml

```diff
@@ -40,12 +39,14 @@ modules:
       - ./waf configure --prefix=${FLATPAK_DEST}
       - ./waf build
       - ./waf install
-      - install -vD --mode 644 data/org.gnome.Hamster.GUI.metainfo.xml ${FLATPAK_DEST}/share/metainfo/org.gnome.Hamster.metainfo.xml
+      - install -vD --mode 644 data/org.gnome.Hamster.GUI.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
     sources:
       - type: git
         url: https://github.com/projecthamster/hamster.git
         commit: 70e2547e7b155bda76ceeeb92f26af5bc177fb0a
       - type: patch
         path: patches/00-fix-missing-tags.patch
-      - type: patch
-        path: patches/01-add-remote-icon.patch
+      - type: file
+        path: org.gnome.Hamster.GUI.png
```

Comment:
just use the svg they bundle, this will look bad

Reply:
Hmm I didn't notice the svg icon. I'll change it the new PR

Reply:
Done in https://github.com/flathub/org.gnome.Hamster/pull/9/commits/0fc475791c6afd7c219f83b82b6283d64c420888

Reply:
Thanks!